acceptance/bundle/artifacts/whl_multiple/output.txt

```diff
@@ -0,0 +1,42 @@
+
+>>> [CLI] bundle deploy
+Building my_test_code...
+Building my_test_code_2...
+Deploying resources...
+Deployment complete!
+Updating deployment state...
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/python-wheel/default/files...
+Uploading my_test_code-0.0.1-py3-none-any.whl...
```

Comment:
The order is a bit confusing now, shall we do the package name replacement instead in these lines and then the output will be determenistic?

Reply:
I agree it's not ideal. I see this as -- we record a set of steps to be taken, ignoring the order.

> do the package name replacement 

I'd rather see the names, given that we focus on multiple wheels being supported there. If we replace both, we won't catch situation if it uploaded one wheel twice instead of two separate wheels once.